lib/signaling/v2/peerconnection.js

```diff
@@ -817,6 +845,7 @@ class PeerConnectionV2 extends StateMachine {
             updateLocalCodecs(this);
             updateRemoteCodecMaps(this);
           }
+          negotiationCompleted(this);
```

Comment:
@makarandp0 If we enhance `negotiationCompleted` like I've suggested then, this will simplify to:
```js
...
else if (description.type === 'answer') {
  this._lastStableDescriptionRevision = this._descriptionRevision;
  negotiationCompleted(this);
}
```